testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AutoTestedSamplesUtil.groovy

```diff
@@ -27,17 +28,16 @@ class AutoTestedSamplesUtil {
     private static final Pattern LEADING_ASTERISK_PATTERN = Pattern.compile(/(?m)^\s*?\*/)
     private static final Pattern LITERAL_PATTERN = Pattern.compile(/\{@literal ([^}]+)}/)
 
-    String includes = '**/*.groovy **/*.java'
-
     void findSamples(String dir, Closure runner) {
         def sources = findDir(dir)
-        def ant = new AntBuilder()
-
-        def list = ant.fileScanner {
-            fileset(dir: sources, includes: includes)
-        }
 
-        list.each() { runSamplesFromFile(it, runner) }
+        Files.walk(Paths.get(sources))
```

Comment:
I think this may implicitly skip some other things because Ant's filescanner has a set of default excludes.

Did this cause any strange failures?

Reply:
Don't think so -- we now run our tests against a few new samples which were previously not executed